### ISSUE TITLE
UUID field doesn't showing

### ISSUE THREAD

**Issue by abisewski** [bn-digital/strapi#10]
Hi!

Even after re-build project the custom field doesn't showing. Any idea the reason about it?

Strapi: 4.6.2
Node Version: 16.16.0

<img width="800" alt="image" src="https://user-images.githubusercontent.com/22551657/228273649-92c34d52-75f6-43fe-992a-588f41388ab1.png">

<img width="362" alt="image" src="https://user-images.githubusercontent.com/22551657/228273549-7e629b92-46d0-4858-96ea-9e750ae1d71e.png">

**Comment by abisewski** [comment]
Actually that was some cache or something like that. After I tried on anonymous page it works. 

But now I'm getting this error clicking on "finish"/"add another field" button

<img width="837" alt="image" src="https://user-images.githubusercontent.com/22551657/228278782-e806d9dd-047a-4382-8e78-c4da3510007b.png">

<img width="818" alt="image" src="https://user-images.githubusercontent.com/22551657/228279171-6a35b681-97f0-4837-99ed-f68fd3d10f6a.png">

**Comment by AlphaYouss** [comment]
Hey, I have the exact same issue you have. The error is the same even if I open it in incognito and disable the cache. Do you have any updates?

**Comment by AlphaYouss** [comment]
Downgrading the package won't help, unfortunately.

**Comment by abisewski** [comment]
No... I dont have any updates :/

**Comment by AlphaYouss** [comment]
@VladyslavVolkov @remidej could one of you look into this?

**Comment by AlphaYouss** [comment]
@abisewski did you try to rebuild your Strapi environment with the npm run build or yarn build? I thought that was unnecessary because I thought that the normal autorun commando npm run develop or yarn develop will do it too, apparently that's not the case. Give that a try, it should work after that.

![chrome-capture-2023-2-31](https://user-images.githubusercontent.com/25977720/229053208-645b94cb-1e71-477d-b4e0-2e7e3e561e1a.gif)

**Comment by abisewski** [comment]
@AlphaYouss Yeah, I did that but didn't work to me... Which Strapi version are you using?

**Comment by AlphaYouss** [comment]
I'm on Strapi version 4.9.0 and Node version 16.19.1.

**Comment by EstebanPiacentino** [comment]
Same issue here. It seems to not be working in 4.9.0

**Comment by Ali-Dalal** [comment]
I also have the same issue

**Comment by sedrickcz** [comment]
Hey. I have the same issue. A downgrade to v1.1.7 helped.

**Comment by Jonashr** [comment]
> Hey. I have the same issue. A downgrade to v1.1.7 helped.

What version of Strapi are you using ? Still can't create a new uuid field downgrading to v1.1.7.

**Comment by soranoo** [comment]
I had the same issue before I updated to Strapi v4.11.0. Simply update Strapi and rebuild the admin panel.

**Comment by Dalmangyi** [comment]
I always typed the “yarn add” command first.
And when I ran the "npm run dev" command, the UUID field did not appear.

I ran the following order: "yarn add" -> "npm run build" -> "npm run dev".
And when I returned to the admin panel, I was able to select it successfully.

I don't even know why...
